site/en/codelabs/openthread-border-router-nat64/index.lab.md

```diff
@@ -65,54 +65,38 @@ The network topology for this codelab:
 Duration: 07:00
 
 
-Follow the  [Setup OTBR step of the Thread Border Router - Bidirectional IPv6 Connectivity and DNS-Based Service Discovery codelab](https://openthread.io/codelabs/openthread-border-router#1) to build the OpenThread border router, with the following change:
+The quickest way to set up an OTBR is by using Docker following the  [OTBR with Docker Guide](https://openthread.io/guides/border-router/docker).
 
-In **Build and install OTBR**, you need to tell the script to enable the NAT64 translator in OpenThread by setting the environment variable `NAT64` to `1` and `NAT64_SERVICE` to `openthread`. Run the following command before the step:
+After OTBR setup is complete, use `ot-ctl` to validate that the NAT64 service is enabled on the border router:
 
 ```console
-$ export NAT64=1 NAT64_SERVICE=openthread
-```
-
-Continue with the  [Thread Border Router - Bidirectional IPv6 Connectivity and DNS-Based Service Discovery](https://openthread.io/codelabs/openthread-border-router#1) codelab as written. After **Form a Thread network**, you can verify the border router is publishing a NAT64 prefix by the OpenThread CLI commands.
-
-Firstly, ensure our border router is up and running, and NAT64 is enabled on the border router:
-
-```console
-$ sudo ot-ctl state
-leader
-Done
-$ sudo ot-ctl nat64 enable
-Done
-$ sudo ot-ctl nat64 state
+$ nat64 state
```

Comment:
Should this be in an OpenThread CLI?

Reply:
Yes, this should be OpenThread CLI, but accessed using `ot-ctl` (as mentioned above).

Reply:
Yes all good, I forgot what it was exactly.